.github/actions/cdn_code_deploy/action.yaml

```diff
@@ -0,0 +1,86 @@
+name: CDN Code Deploy 
+description: Action that Deploy an artifact to the CDN
+
+# This action allows you to deploy the passed artifact to the CDN.
+# The artifact must be the directory that contains all the files you want to sync.
+
+inputs:
+  storage_account_name:
+    description: Storage account name.
+    type: string
+    required: true
+  blob_container_name:
+    description: Blob container name.
+    type: string
+    required: true
+  resource_group_name:
+    description: CDN resource group name.
+    type: string
+    required: true
+  profile_name:
+    description: CDN Profile name.
+    type: string
+    required: true
+  endpoint_name:
+    description: CDN endpoint name.
+    type: string
+    required: true
+  selective_purge_paths:
+    description: |
```

Comment:
does this supports glob patterns / dir or only single files?

Reply:
if you want to point to a whole directory you have to end the path with `/*` because the purge always refers to files